app/lib/clients/vault/policy.rb

```diff
@@ -1,12 +1,23 @@
 module Clients
   class Vault
     module Policy
+      extend Entity
+
       def rotate_token
         create_astral_policy
         token = create_astral_token
         Clients::Vault.token = token
       end
 
+      def assign_policy(identity, policy_path)
+        sub = identity.sub
+        email = identity.email
+        policies, metadata = get_entity_data(sub)
+        policies.append(policy_path).to_set.to_a
```

Comment:
I think this could be:
```
        policies.append(policy_path).uniq!
```

I was going to try it in my PR but forgot

Reply:
`to_set` will uniquify

Reply:
but i see, we are not capturing in that case back to `policies` -- made your suggested change